src/main/java/dev/openfeature/javasdk/FlagEvaluationDetails.java

```diff
@@ -14,8 +14,9 @@
     private String flagKey;
     private T value;
     @Nullable private String variant;
-    private Reason reason;
-    @Nullable private String errorCode;
+    @Nullable private String reason;
+    private ErrorCode errorCode;
+    @Nullable private String message;
```

Comment:
Spec change.

Reply:
I think I missed this. We have both `reason` and `message`? Why would you want both?

Reply:
Reason is always present. It's the semantic reason for a flag evaluation. `message` is literally the `message` property of a thrown exception. You can see we populate it with `e.message` in the handler.

I think it was one of the more popular parts of this change because actual error messages were being swallowed. It's quite useful for debugging.

Reply:
I've updated this to `errorMessage`